scripts/qa_crowbarsetup.sh

```diff
@@ -1747,7 +1753,6 @@ function reboot_nodes_via_ipmi
                 "default gateway to be active in bmc"
 
             $ipmicmd chassis bootdev pxe options=persistent
-            $ipmicmd mc reset warm
```

Comment:
The mc reset was removed and tested on several qa hardware, and so far no issues with ipmi.
The next wait_for return immediately. So no harm done i guess.